### PR TITLE
New package: lf-r3

### PR DESCRIPTION
**Pull request by TeddyDD**
lf is terminal file manager similar to ranger but simpler
https://github.com/gokcehan/lf

### CODE REVIEW

File: srcpkgs/lf/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'lf'
+pkgname=lf
+version=1
```
Comment (cr6git) [inline srcpkgs/lf/template]: `version=r1`

Comment (TeddyDD) [reply srcpkgs/lf/template]: So version does not have to be a number? Okay :)

File: srcpkgs/lf/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+revision=1
+build_style=go
+go_import_path="github.com/gokcehan/${pkgname}"
+wrksrc="${pkgname}-r${version}"
```
Comment (cr6git) [inline srcpkgs/lf/template]: then you can omit `wrksrc="...`

File: srcpkgs/lf/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+hostmakedepends="git"
+short_desc="Terminal file manager"
+maintainer="Daniel Lewan <vision360.daniel@gmail.com>"
+license="MIT"
+homepage="https://${go_import_path}"
```
Comment (cr6git) [inline srcpkgs/lf/template]: `homepage="https://godoc.org/github.com/gokcehan/lf`

File: srcpkgs/lf/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+distfiles="${homepage}/archive/r${version}.tar.gz"
```
Comment (cr6git) [inline srcpkgs/lf/template]: distfiles=`https://github.com/gokcehan/lf/archive/${version}.tar.gz`

File: srcpkgs/lf/template (hunk 1 above)
Comment (cr6git) [reply srcpkgs/lf/template]: version is what upstream provides.

File: srcpkgs/lf/template (hunk 3 above)
Comment (cr6git) [reply srcpkgs/lf/template]: change that.

File: srcpkgs/lf/template (hunk 4 above)
Comment (cr6git) [reply srcpkgs/lf/template]: change that.

File: srcpkgs/lf/template (hunk 3 above)
Comment (TeddyDD) [reply srcpkgs/lf/template]: Not sure about that, readme in repo is more informative, has screenshots, links to community chat and list of features.

File: srcpkgs/lf/template (hunk 1 above)
Comment (TeddyDD) [reply srcpkgs/lf/template]: Okay, thanks. I saw some packages omniting "v" from versions number and assumed this should be stripped too..

**Comment by TeddyDD** [comment]
@cr6git I'd prefer not to change homepage. Let me know if I should untalgle distfile and homepage into literal adress, without varialbes.

**Comment by cr6git** [comment]
@TeddyDD don't worry, maybe someday a developer will merge your stuff.

**Comment by TeddyDD** [comment]
Three updates so far :rofl:

**Comment by pullmoll** [comment]
15e9121cc826c34459f34b287323c57f37146a9f

**Pull request closed without merging**